code/core/src/manager/components/preview/tools/zoom.tsx

```diff
@@ -40,20 +41,26 @@ const Zoom = memo<{
   reset: MouseEventHandler;
 }>(function Zoom({ zoomIn, zoomOut, reset }) {
   return (
-    <>
-      {/* @ts-expect-error (non strict) */}
-      <IconButton key="zoomin" onClick={zoomIn} title="Zoom in">
-        <ZoomIcon />
-      </IconButton>
-      {/* @ts-expect-error (non strict) */}
-      <IconButton key="zoomout" onClick={zoomOut} title="Zoom out">
-        <ZoomOutIcon />
-      </IconButton>
-      {/* @ts-expect-error (non strict) */}
-      <IconButton key="zoomreset" onClick={reset} title="Reset zoom">
-        <ZoomResetIcon />
-      </IconButton>
-    </>
+    <ToolList>
+      <li>
+        {/* @ts-expect-error (non strict) */}
+        <IconButton key="zoomin" onClick={zoomIn} title="Zoom in">
+          <ZoomIcon />
+        </IconButton>
+      </li>
+      <li>
+        {/* @ts-expect-error (non strict) */}
+        <IconButton key="zoomout" onClick={zoomOut} title="Zoom out">
+          <ZoomOutIcon />
+        </IconButton>
+      </li>
+      <li>
+        {/* @ts-expect-error (non strict) */}
+        <IconButton key="zoomreset" onClick={reset} title="Reset zoom">
+          <ZoomResetIcon />
+        </IconButton>
+      </li>
+    </ToolList>
```

Comment:
Didn't you already wrap those over here already?
https://github.com/storybookjs/storybook/blob/65a6d76bd7cd7adf3bde10a85af5d664afad785e/code/core/src/manager/components/preview/Toolbar.tsx#L185-L190

Reply:
For this issue, I think that each button should be wappred with `<li>`, but `Zoom` component includes multiple buttons and so three buttons are grouped by only one `<li>` tag. So I nested unordered list.

Reply:
For this issue, I think that each button should be wappred with `<li>`, but `Zoom` component includes multiple buttons and so three buttons are grouped by only one `<li>` tag. So I nested unordered list(`manager.tsx` is the same).